ReleaseHistory.md

```diff
@@ -2,11 +2,12 @@
 
 ## 4.6.0 UNRELEASED
 * BRK: Remove defunct and unsupported `kusto` command in `Sarif.Multitool`.
+* BRK: Remove support for .NET Core 3.1 and .NET 6.0. Use a supported version of .NET: https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core
```

Comment:
>NET [](http://example.com/codeflow?start=45&length=3)

should we have an explicit reference to our use of net8.0? oner aspect of this breaking change for the multitool, for example, is the the tools directory in the package now references this version.

Reply:
For god's sake. It's impossible to work with me.

BRK: Remove support for .NET Core 3.1 and .NET 6.0 in preference of a [supported version of .NET](https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core), `net8.0`.

Reply:
No worries and done. 😊